src/main/java/com/configcat/ConfigCatClient.java

```diff
@@ -537,31 +553,32 @@ public static ConfigCatClient get(final String sdkKey) {
     }
 
     /**
-     * Get a singleton ConfigCat Client for the sdk key with the options, create a new client if not existed yet.
-     * If the client already exist the options are ignored.
+     * Creates a new or gets an already existing ConfigCatClient for the given sdkKey.
      *
-     * @param sdkKey  the client sdk key.
-     * @param options the client initializer options.
-     * @return a singleton client.
+     * @param sdkKey          the SDK Key for to communicate with the ConfigCat services.
+     * @param optionsCallback the options callback to configure the created ConfigCatClient instance.
```

Comment:
We should avoid the word `configure` and `config` in all documentation because it collides with the configuration service expression and the configs we provide.